pages/common/doctl-databases-pool.md

```diff
@@ -0,0 +1,24 @@
+# doctl databases pool
+
+> Manage connection pools for your database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool/>.
```

Comment:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool>.
```

Reply:
I am resolving this suggestion as the site resolves to the address with `/`. We only remove it in cases when the site doesn't redirect to a URL with `/`.